edisgo/grid/components.py

```diff
@@ -254,6 +257,65 @@ def peak_load(self):
 
         return peak_load
 
+    @property
+    def reactive_power_mode(self):
+        """
+        Power factor mode of Load.
+
+        If the power factor is set, then it is necessary to know whether the
+        it is leading or lagging. In other words this information is necessary
+        to make the generator behave in an inductive or capacitive manner.
+        Essentially this changes the sign of the reactive power Q.
+
+        The convention used here in a load is that:
+        - when `reactive_power_mode` is 'inductive' then Q is positive
+        - when `reactive_power_mode` is 'capacitive' then Q is negative
+
+        In the case that this attribute is not set, it is retrieved from the
+        network config object depending on the voltage level the load is in.
+
+        Returns
+        -------
+        :obj: `str` : Power factor mode
+            Either 'inductive' or 'capacitive'
+        """
+        if self._reactive_power_mode is None:
+            if isinstance(self.grid, MVGrid):
+                self._reactive_power_mode = self.grid.network.config[
+                    'reactive_power_mode']['mv_load']
+            elif isinstance(self.grid, LVGrid):
+                self._reactive_power_mode = self.grid.network.config[
+                    'reactive_power_mode']['lv_load']
+
+        return self._reactive_power_mode[1:-1]
+
+    @reactive_power_mode.setter
+    def reactive_power_mode(self, reactive_power_mode):
+        """
+        Set the power factor mode of the generator.
+        Should be either 'inductive' or 'capacitive'
+        """
+        self._reactive_power_mode = reactive_power_mode
+
+    @property
+    def q_sign(self):
+        """
+        Get the sign reactive power based on the
+        :attr: `_reactive_power_mode`
+
+        Returns
+        -------
+        :obj: `int` : +1 or -1
+        """
+        comparestr = self.reactive_power_mode
+        comparestr = comparestr.lower()
+        if re.fullmatch('inductive', comparestr):
```

Comment:
You can just do `if self.reactive_power_mode == 'inductive'`

Reply:
The idea was to use a regex match as a secure and clear check, I can change this.